homeassistant/components/cloud/http_api.py

```diff
@@ -206,19 +221,53 @@ async def post(self, request: web.Request) -> web.Response:
 class CloudLoginView(HomeAssistantView):
     """Login to Home Assistant cloud."""
 
+    _mfa_tokens: dict[str, str] = {}
+    _mfa_tokens_set_time: float = 0
+
     url = "/api/cloud/login"
     name = "api:cloud:login"
 
     @require_admin
     @_handle_cloud_errors
     @RequestDataValidator(
-        vol.Schema({vol.Required("email"): str, vol.Required("password"): str})
+        vol.Schema(
+            {
+                vol.Required("email"): str,
+                vol.Required(vol.Any("password", "code")): str,
```

Comment:
I think this means that both password and code can be passed at the same time. If that's ok we'll need to update the tests to allow that.

Reply:
Yeah, I did not think of that case. I updated the validation to allow only one of them. This works as expected, do you think it also makes sense?